### PR TITLE
[IOTDB-959]Add createStorageGroup Grammar

### PR DESCRIPTION
**Pull request by ijihang**
JR:https://issues.apache.org/jira/browse/IOTDB-959

### CODE REVIEW

File: antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4 [hunk 1]
```diff
@@ -110,6 +110,7 @@ statement
       cqSelectIntoClause #createContinuousQueryStatement
     | DROP (CONTINUOUS QUERY | CQ) continuousQueryName=ID #dropContinuousQueryStatement
     | SHOW (CONTINUOUS QUERIES | CQS) #showContinuousQueriesStatement
+    | CREATE STORAGE GROUP prefixPath #createStorageGroup
```
Comment (Alima777) [inline antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4:113]: Move this next to `SET STORAGE GROUP ...` to put the same type command together.

**Review by Alima777 — COMMENTED** [review]
Add an integration test, execute `create storage group` command and `show storage group` command to check the result.

File: antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4 (hunk 1 above)
Comment (ijihang) [reply antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlBase.g4]: Thanks, new commit add integration test and move sql

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    for (String storageGroup : storageGroups) {
+      statement.execute(String.format("create storage group %s", storageGroup));
+    }
+
+    // ensure that current StorageGroup in cache is right.
+    createStorageGroupTool(storageGroups);
+
+    EnvironmentUtils.stopDaemon();
+    setUp();
```
Comment (Alima777) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java:77]: Why a recover? It seems the following test you exectued does not need that.

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+        String storageGroupPath = resultSet.getString("storage group");
+        resultList.add(storageGroupPath);
+      }
+    }
+    Assert.assertEquals(3, resultList.size());
+
+    List<String> collect =
+        resultList.stream()
+            .sorted(Comparator.comparingInt(e -> e.split("\\.").length))
+            .collect(Collectors.toList());
```
Comment (Alima777) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java:99]: `resultList = resultList.stream().sorted().collect(Collectors.toList());` is ok

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java (hunk 2 above)
Comment (ijihang) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java:77]: Test whether the storage group still exists when the iotdb is stopped and then restarted

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java (hunk 3 above)
Comment (ijihang) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java:99]: yes

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java (hunk 2 above)
Comment (Alima777) [reply server/src/test/java/org/apache/iotdb/db/integration/IoTDBCreateStorageGroupIT.java:76]: ok :D

**Review by Alima777 — APPROVED** [review]

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**